src/Core/src/Layouts/Flex.cs

```diff
@@ -392,41 +391,30 @@ public Item(float width, float height)
 			Height = height;
 		}
 
-		public void Add(Item child)
+		public new void Add(Item child)
```

Comment:
Inherit from List and uses the `new` on base methods is dangerous, if at some part of the code `Item` is cast to `List<Item>`, just the `base.Add` will be executed (The same for `InsertAt`). I believe having this class inherit from `IEnumerable` is to avoid this pitfall.

Reply:
That is a fair point. Although, `Flex.Item` is an internal class and I don't see anywhere in the codebase that it is being cast to a list.

Probably some of the same benefits could be attained by just figuring out how to remove the FlexExtensions file and simply using the built-in list functions of the `Children` property directly.

Although, this approach also simplifies the code a bit.